packages/core/src/Stores/Modules/Trading/Helpers/active-symbols.js

```diff
@@ -37,19 +37,23 @@ export const pickDefaultSymbol = (active_symbols = []) => {
 };
 
 const getFavoriteOpenSymbol = (active_symbols) => {
-    const chart_favorites = LocalStore.get('cq-favorites');
-    if (!chart_favorites) return undefined;
-    const client_favorite_markets = JSON.parse(chart_favorites)['chartTitle&Comparison'];
+    try {
+        const chart_favorites = LocalStore.get('cq-favorites');
+        if (!chart_favorites) return undefined;
+        const client_favorite_markets = JSON.parse(chart_favorites)['chartTitle&Comparison'];
 
-    const client_favorite_list = client_favorite_markets
-        .map(client_fav_symbol => active_symbols
-            .find(symbol_info => symbol_info.symbol === client_fav_symbol));
-    if (client_favorite_list) {
-        const client_first_open_symbol = client_favorite_list
-            .filter(symbol => symbol).find(isSymbolOpen);
-        if (client_first_open_symbol) return client_first_open_symbol.symbol;
+        const client_favorite_list = client_favorite_markets
+            .map(client_fav_symbol => active_symbols
+                .find(symbol_info => symbol_info.symbol === client_fav_symbol));
+        if (client_favorite_list) {
+            const client_first_open_symbol = client_favorite_list
+                .filter(symbol => symbol).find(isSymbolOpen);
+            if (client_first_open_symbol) return client_first_open_symbol.symbol;
+        }
+        return undefined;
+    } catch (error) {
+        return undefined;
```

Comment:
What error is going to be thrown by LocalStore.get? can we comment that in the code?

Reply:
It's getting user's favorite market, so we can not comment that

Reply:
The error that QA reported, should be handled some how

Reply:
@aliafsah-fs Please add the comment for this.

Reply:
@reza-fs @mahdipourziaei-binary comments added